src/main/java/com/gregtechceu/gtceu/common/data/GTRecipes.java

```diff
@@ -14,11 +16,24 @@
 
 import it.unimi.dsi.fastutil.objects.ObjectOpenHashSet;
 
+import java.util.EnumMap;
+import java.util.Map;
 import java.util.Set;
 import java.util.function.Consumer;
 
 public class GTRecipes {
 
+    public static final Map<BlastProperty.GasTier, FluidIngredient> EBF_GASES = new EnumMap<>(
+            BlastProperty.GasTier.class);
+
+    static {
+        EBF_GASES.put(BlastProperty.GasTier.LOW, FluidIngredient.of(1000, GTMaterials.Nitrogen.getFluid()));
+        EBF_GASES.put(BlastProperty.GasTier.MID, FluidIngredient.of(100, GTMaterials.Helium.getFluid()));
+        EBF_GASES.put(BlastProperty.GasTier.HIGH, FluidIngredient.of(50, GTMaterials.Argon.getFluid()));
+        EBF_GASES.put(BlastProperty.GasTier.HIGHER, FluidIngredient.of(25, GTMaterials.Neon.getFluid()));
+        EBF_GASES.put(BlastProperty.GasTier.HIGHEST, FluidIngredient.of(10, GTMaterials.Krypton.getFluid()));
```

Comment:
these should actually be changed to use tag ingredients now that you've touched them.